### PR TITLE
Tests/move linear model

### PR DESCRIPTION
**Pull request by aqitya**
Improvements to linear_model_tests. Moved linear model tests from test_base_models to a separate file. This PR takes a step towards implementing a fix for #191.

### CODE REVIEW

File: tests/test_linear_model.py [hunk 1]
```diff
@@ -0,0 +1,297 @@
+# Copyright © 2021 United States Government as represented by the Administrator of the National Aeronautics and Space Administration.  All Rights Reserved.
+
+import numpy as np
+import unittest
+
+from prog_models import *
```
Comment (teubert) [inline tests/test_linear_model.py:6]:
```suggestion
```
Looking through the code, I don't think that line is needed

**Review by teubert — CHANGES_REQUESTED** [review]
Great update that will set the stage for future expansion of the linear model tests. 

A few small suggestions- one to fix the issue with the automated tests, and the second to clean up unused imports

Great work!

File: tests/test_linear_model.py [hunk 2]
```diff
@@ -0,0 +1,297 @@
...
+from prog_models.models.test_models.linear_models import FNoneNoEventStateLM
+from test_base_models import LinearThrownObject
```
[the remark this answers is not in the document]
Comment (aqitya) [reply tests/test_linear_model.py]: Yep, makes sense. Will do the corresponding changes promptly.

**Review by teubert — APPROVED** [review]
Looks good- good work. 

One last comment: I suggest you remove LinearThrownObject from test_base_models as well, and replace it with an import from test_models.linear_thrown_object

**Review by kjjarvis — APPROVED** [review]
Good start to improving linear_tests. New formatting looks good

**Pull request merged**